morse.h

```diff
@@ -233,11 +233,20 @@ class SpeakerMorseSender: public MorseSender {
  * Sends Morse on a digital output pin.
  */
 class LEDMorseSender: public MorseSender {
+	private:
+		bool activeLow;
 	protected:
 		virtual void setOn();
 		virtual void setOff();
 	public:
-		LEDMorseSender(int outputPin, float wpm=WPM_DEFAULT);
+		/**
+		 * Creates a LED Morse code sender with the given GPIO pin.  The optional
+		 * boolean activeLow indicates LED is ON with digital LOW value.
```

Comment:
Thanks for adding a doc too! I think this line is covered by the @param below, so it's OK to omit so there's not duplication.